File: Listas de Exercícios (Python)/Python/Lista 2 - Sala/2.py
print("Aprovação - Aluno\n")

print("--Valores decimais devem ser digitados com (.)!--")

N1 = float(input("Nota da primeira prova: "))
N2 = float(input("Nota da segunda prova: "))

M = (N1+N2)/2

print("\n==================")
print (f"Média inicial: {M}")
print("==================")

print("")

if (M>=6.0):
    print("Parabéns!!\nVocê foi aprovado!")
else:
    E = int(input("Nota do exame (faremos uma nova média com essa nota): "))

    M = (M+E)/2

    print("\n=================")
    print(f"Média final: {M}")
    print("=================")

    print("")
    if (M>=6.0):
        print("Parabéns!!\nVocê foi aprovado!")
    else:
        print("Você foi reprovado!")

print("")

File: Lista de Exercícios (C)/C/Lista 2 - Sala/2.c
#include <stdio.h>

int main()
{
    float M, N1, N2, E;
    
    printf ("Aprovação - Aluno\n\n");
    
    printf("Nota da primeira prova: "); scanf("%f", &N1);
    printf("Nota da segunda prova: "); scanf("%f", &N2);
    
    M=(N1+N2)/2;
    
    printf("\n==================\n");
    printf("Média inicial: %.2f", M);
    printf("\n==================\n\n");
    
    if (M>=6.0) {
        printf("Parabens!!\nVocê foi aprovado!");
    }
    else {
        printf("Nota do exame (faremos uma nova média com essa nota): "); scanf("%f", &E);
        
        M=(M+E)/2;
        
        printf("\n==================\n");
        printf("Média final: %.2f", M);
        printf("\n==================\n\n");
        
        if (M>=6.0) {
            printf("Parabéns!!\nVocê foi aprovado!");
        }
        else {
            printf("Você foi reprovado!");
        }
    }
    }